primefaces-integration-tests/src/test/java/org/primefaces/integrationtests/confirmdialog/ConfirmDialog001Test.java

```diff
@@ -47,6 +47,7 @@ class ConfirmDialog001Test extends AbstractPrimePageTest {
     void showDialog(Page page) {
         // Arrange
         ConfirmDialog dialog = page.dialog;
+        noMinLoadAnim();
```

Comment:
Why is this required? Makes the tests a bit hacky

Reply:
ConfirmDialog tests checked if buttons were enabled while they were not (yet) due to the minimum loading time.

Reply:
you mean the assertClickable(page.confirm);?

maybe we should instead improve this and check the animination state and wait for it

Reply:
Alright. I'll also add some additional tests to the IT I've added.